docs/README.md

```diff
@@ -20,8 +20,10 @@ installed. Also install the following libraries:
     $ sudo pip install Pygments
     # Following is needed only for generating API docs
     $ sudo pip install sphinx
-    $ Rscript -e 'install.packages(c("knitr", "devtools"), repos="http://cran.stat.ucla.edu/")'
+    $ sudo Rscript -e 'install.packages(c("knitr", "devtools", "roxygen2", "testthat"), repos="http://cran.stat.ucla.edu/")'
```

Comment:
So `install.packages` for R should work even without `sudo`. It will install packages in the user's R lib directory instead of the system wide one. 

That said I don't mind using `sudo` if we want things to be uniform in this doc.